### ISSUE TITLE
How to eliminate this warning?

### ISSUE THREAD

**Issue by chenzhao17** [wang-xinyu/tensorrtx#225]
Thanks again for this wonderful project. I succeeded in converting yolov5 model using this repo, but the occurrence of the following warning during compilation is quite annoying. Could somebody instruct me on how to eliminate it?

tensorrtx/yolov5/yololayer.h(85): warning: function "nvinfer1::IPluginV2Ext::configurePlugin(const nvinfer1::Dims *, int, const nvinfer1::Dims *, int, const nvinfer1::DataType *, const nvinfer1::DataType *, const __nv_bool *, const __nv_bool *, nvinfer1::PluginFormat, int)" is hidden by "nvinfer1::YoloLayerPlugin::configurePlugin" -- virtual function override intended?

**Comment by samiit** [comment]
I am getting the same error.

```
/home/Projects/tensorrtx/yolov5/yololayer.h(103): warning: function "nvinfer1::IPluginV2Ext::configurePlugin(const nvinfer1::Dims *, int, const nvinfer1::Dims *, int, const nvinfer1::DataType *, const nvinfer1::DataType *, const __nv_bool *, const __nv_bool *, nvinfer1::PluginFormat, int)" is hidden by "nvinfer1::YoloLayerPlugin::configurePlugin" -- virtual function override intended?

/home/Projects/tensorrtx/yolov5/yololayer.cu(9): error: no instance of overloaded function "nvinfer1::YoloLayerPlugin::YoloLayerPlugin" matches the specified type
...

```
There are still a error statements that follow, but I am only including the first.

Looking forward to some help.

**Comment by EchizenG** [comment]
I also have this

**Comment by henbucuoshanghai** [comment]
CUDA error 222 at tensorrtx/yolov4/yololayer.cu:19yolov4: ensorrtx/yolov4/yololayer.cu:19: nvinfer1::YoloLayerPlugin::YoloLayerPlugin(): Assertion `0' failed.
Aborted (core dumped)


I dont konw why    help

**Comment by henbucuoshanghai** [comment]
idden by "nvinfer1::YoloLayerPlugin::configurePlugin" -- virtual function override intended?

have you find the soulution?

**Comment by pshwetank** [comment]
yeah....facing the same warning. No idea what the problem is

**Comment by wang-xinyu** [comment]
This warning is caused by the nvinfer headers, seems cannot be solved under trt 7.0. I don't know if future versions of tensorrt can solve this.
